_pages/faq.md

```diff
@@ -2,126 +2,175 @@
 title: Frequently asked questions
 ---
 
-# Frequently asked questions
+<div class="usa-accordion usa-accordion-bordered" aria-multiselectable="true">
```

Comment:
This page previously wrapped each question in its own accordion — this switches to a single accordion with `aria-multiselectable="true"`, which keeps the same behavior (more than one question can be expanded at the same time).

I’m not sure this is a great use of accordions in general, but that’d be a separate PR!